apps/protocol-frontend/src/utils/validations.ts

```diff
@@ -46,6 +46,7 @@ export const editContributionFormValidation = yup.object({
 export const reportFormValidation = yup.object({
   name: yup.string().required('This field is required.'),
   engagementDate: yup.date(),
+  activityType: yup.string().min(1).required('This field is required.'),
```

Comment:
I think we may be able to just use `yup.string().required(`This field is required.')` since we don't need the `.min(1)` for a minimum length

Reply:
did you get a chance to resolve this @NKoech123 ?

Reply:
Yes sir. Pushed the change a few minutes @jonathanprozzi . Let me know if more edits are required.